code/obj/storage/secure_closets.dm

```diff
@@ -9,12 +9,18 @@
 	name = "personal locker"
 	desc = "The first card swiped gains control."
 	personal = 1
-	spawn_contents = list(/obj/item/storage/backpack,
-	/obj/item/storage/backpack/satchel,
-	/obj/item/device/radio/signaler,
+	spawn_contents = list(/obj/item/device/radio/signaler,
 	/obj/item/pen,
 	/obj/item/device/radio/headset)
 
+	make_my_stuff() //Let's spawn the backpack/satchel in random colours!
+		. = ..()
+		if (. == 1) //if we've not spawned stuff before
```

Comment:
what is `1`

Reply:
A bit of a weird thing:
`make_my_stuff` returns 1 once it's done spawning the stuff it should and 0. AFAIK everything calling it sets spawn_contents to null right after (it doesn't to so automatically). I'm guessing it's that way in case someone wants to do something special.

Should I change it?

Reply:
fine for now.